src/index.js

```diff
@@ -0,0 +1,10 @@
+import './style.css';
+import { Leaderboard } from './scores.js';
+
+const leaderboard = new Leaderboard();
+
+leaderboard.addScore();
+
+window.onload = () => {
+  leaderboard.loadData();
+};
```

Comment:
![image](https://github.com/ulises2607/Leaderboard-list/assets/93039359/fedeb46b-cd01-4254-b149-8c2cc91a7ab8)

- [ ] It would be nice if you fix the issue which I am facing which you can see in the image above ⬆️ It is coming because you don't have any method with the name of `loadData` 😨 but I can see that you have a method called `displayData` Kindly fix it to make your app work as expected 👍 😊